ui.go

```diff
@@ -416,7 +416,7 @@ func (win *win) printDir(screen tcell.Screen, dir *dir, selections map[string]in
 			}
 		}
 
-		if i == dir.pos {
+		if i == dir.pos && gOpts.focus == "on" {
```

Comment:
Minor nit: I'm not sure if "focus" is the right name for this option. At this point, it's an option that chooses whether to draw cursors, and the user decides whether that has anything to do with focus.

Reply:
Maybe instead add a 'cursorfmt' option that takes a format string like 'tagfmt' so you can apply an arbitrary style.

Reply:
I agree, the name is not ideal.

Reply:
I've changed it back to be a bool since ilyagr does not need it anymore and cursor is not quite right (we already have a text cursor).